ipatests/conftest.py

```diff
@@ -71,9 +71,31 @@ def pytest_configure(config):
     config.option.doctestmodules = True
 
 
+def pytest_addoption(parser):
+    def truefalse(arg):
+        if arg.lower() == 'true':
+            return True
+        if arg.lower() == 'false':
+            return False
+        return arg  # triggers an error later
+
+    group = parser.getgroup("IPA integration tests")
+    group.addoption(
+        '--in-server',
+        dest="ipa_in_server",
+        type=truefalse,
+        choices=(True, False),
+        default=False,
```

Comment:
You don't need to go the complicated route like in PR #442. Please replace type, choices and default with a simple ```action='store_true```, https://docs.python.org/3/library/argparse.html#action